### PR TITLE
PWEB-5494 - Add back ability to use a scheme url

### PR DESCRIPTION
**Pull request by neitzj**
(i.e.. ddx://<share url>) for testing the joinSession api

### CODE REVIEW

File: DDx/DDx/AppDelegate.m [hunk 1]
```diff
@@ -59,12 +59,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
             return NO;
         }
     } 
-    else 
-    {
```
Comment (julieiannelli) [inline DDx/DDx/AppDelegate.m]: Let's add documentation telling devs how to use and test both connection methods
We'll also need to update our test cases accordingly

**Comment by neitzj** [comment]
Added documentation next to the area of the code where the joinSession is called

**Comment by kiddlec** [comment]
I suppose I am getting a bit picky - but figure it is best not to refer to a specific version in the comments:

Would recommend changing the line starting with "When you click this link" to:

"When you click this link the PureWeb server will redirect this url to a webpage where you can"

Also - please change "redirect your mobile client" to "redirect to your mobile client"

**Comment by neitzj** [comment]
Done.

**Comment by kiddlec** [comment]
+1

**Comment by julieiannelli** [comment]
+1 cause @kiddlec said so

**Pull request merged**